### PR TITLE
Project work

### PR DESCRIPTION
**Pull request by OscarFMdev**
## PR description

In this branch I:
- Created a mobile version of the webpage.
- Created a desktop version of the webpage.
- Configurated JavaScript file to monitor user interaction and create speaker cards dynamically.
- Fixed linter errors.

**Video:** [Project Video](https://www.loom.com/share/e1de3ba9c22b4a4bb97a07b1d1d71b03)

#### Thank you for reviewing my code! 🙌

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          </div>
+            
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <h6>&copy; International Technologies Conference</h6>
+    </footer>
```
Comment (Bondok6) [inline about.html]: - [ ] According to the original design The footer should contain a logo aligned to the left and text aligned to the right, Kindly update it 👌

| Your Design  | Original Design  |   
| --- | --- |
|   |  |

- [ ] [ in the mobile version/size ] you should change the background color to white  and I suggest adding the `partners section` above the footer section to follow the original design 💯👌

> Original Design
image

File: index.html [hunk 2]
```diff
@@ -1,13 +1,134 @@
...
+        </div>
+        
+      </section>
+      <!-- Speakers section -->
+    </main>
+    <div class="last-section">
+      <footer>
+        <h6>&copy; International Technologies Conference</h6>
+      </footer>
+    </div>
```
Comment (Bondok6) [inline index.html]: - [ ] I think you forgot to add the partners section, kindly add it 👌
- [ ] According to the original design The footer should contain a logo aligned to the left and text aligned to the right, Kindly update it 👌
- [ ] Footer section on the home page: It should have a white background. 💯

| Your Design  | Original Design  |   
| --- | --- |
|   |  |

File: index.html [hunk 3]
```diff
@@ -1,13 +1,134 @@
...
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Create free networking opportunities with ITC staff from around the world.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+
+        </div>
+        <!-- cross bg -->
+      </section>
```
Comment (Bondok6) [inline index.html]: - [ ] Great job here 💯but you I think you forgot to add the button, kindly add it to look similar to the original desig👌

| Your Design  | Original Design  |   
| --- | --- |
|   |  |

- NOTE: this button on the mobile version/size should look like this ⬇

image

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @OscarFMdev 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## To Highlight 
- Great job using semantic tags ✅
- No linter errors ✅
- Your repo is professional ✅
- Your project is professional ✅

## Suggested changes
- [ ] According to the requirements, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

image

- [ ] The Link of the MIT file in your README file is broken, Kindly fix it by changing the path to `This project is [MIT](./MIT.md) licensed.` instead of `This project is [MIT](./LICENSE) licensed.`  or you can rename your `MIT.md` to `LICENSE` 💯

- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/scss/layout/_mobile-menu.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
+@import '../base/variables';
+
+.mobile-menu {
```
Comment (tufoinnkuo10) [inline src/scss/layout/_mobile-menu.scss:3]: - [optional] please ensure that when the mobile popup comes up it should not be possible to scroll through your site. great job

File: index.html [hunk 5]
```diff
@@ -1,13 +1,149 @@
...
+              <p class="program-desc">You can create your own creations using open source code instead of just looking at them.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Create free networking opportunities with ITC staff from around the world.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+          <button class="program-button"><a class="program-link" href="#">See the whole program</a></button>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please add the `see the whole program` link on the desktop version
- [ ] kindly use the same color and design for the `see the whole program` button on the mobile version

your version
Screenshot (455)_LI
original version
Screenshot (454)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @OscarFMdev 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ followed the design
✅ Menu ok

Required Changes ♻️
Check the comments under the review
Optional suggestions
- [optional] please add a good brief description in your readme on what your project is all about.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by OscarFMdev** [comment]
Thank you @Bondok6 for your feedback! it was complete and clear.

**Review by Kossi-stack — COMMENTED** [review]
## Status: Approved ✔️

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @OscarFMdev,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**